content/community/projects.md

```diff
@@ -78,11 +78,9 @@ The official Bluesky app is available on the [iOS App](https://apps.apple.com/us
 - [Klearsky](https://klearsky.pages.dev/) by [@mimonelu.net](https://bsky.app/profile/did:plc:ilxxgyz7oz7mysber4omeqrg)
 - [Laika](https://laika-bluesky.web.app/) by [@makoto.bsky.social](https://bsky.app/profile/did:plc:tr5cijtwcpl2dqrtmjsmmcow)
 - [Skylight](https://penpenpng.github.io/skylight) by [@penpenpng.bsky.social](https://bsky.app/profile/did:plc:ahj2yuo63gaxyhbgn5ai6jt2)
-- [The Blue](https://the-blue.shino3.net) by [@shino3.bsky.social](https://bsky.app/profile/did:plc:rpwpuzu2yyiuufm3232d7pm5)
```

Comment:
why is this one removed? seems to still be functional to me

Reply:
![Screenshot_2023-09-23-05-26-17-45_984e1414ae90666a90f12ff17ec14a7f](https://github.com/bluesky-social/atproto-website/assets/20620901/0bec4bf3-0e10-4dd8-bda9-7c7e29ccae9f)

I'm not sure if this is indicative of being usable, it's a farewell message.

Reply:
ah thanks for the screenshot! the client loads a page that allows me to login on first click (screenshot below), but switches to the "see you again" screen on refresh. going to optimistically attribute this to a bug for now, but we can revisit this link after some time to see if it's still functional 

![Screen Shot 2023-09-22 at 4 07 26 PM](https://github.com/bluesky-social/atproto-website/assets/22510954/bd9525bd-3ae4-4e21-b917-85049819ce43)

mind putting this line back? then i'll approve the pr 👍  thanks for reviewing all of the clients! 🙏

Reply:
very likely that there might be some service worker caching going and the new "farewell page" replacing it didn't replace the SW as well, causing the stale cache to be loaded instead

I'll come around to this PR in a an hour or two

Reply:
should be good to go